<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cloud</class>
 <widget class="QDialog" name="cloud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>离线空间</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>931</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>文件列表</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>931</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2">
    <attribute name="title">
     <string>正在下载</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>921</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>正在上传</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>921</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
